src/wasm-binary.h

```diff
@@ -863,6 +863,7 @@ class WasmBinaryBuilder {
     BreakTarget(Name name, int arity) : name(name), arity(arity) {}
   };
   std::vector<BreakTarget> breakStack;
+  std::unordered_set<Name> seenBreaks;
```

Comment:
How about calling this `breakTargets` instead, since it's not a set of the break instructions, but rather their targets?

Reply:
Hmm, maybe, but we have a struct called `BreakTarget` so that might be confusing. It contains a name and and arity. So maybe `breakTargetNames`?

Reply:
Sounds good.

Reply:
Done.